### PR TITLE
Removed Bias neuron from last layer

### LINKED ISSUE #63: Network architectures include bias neuron in the last layer
According to our theory, networks trained with entropic open-set loss and objectosphere should not include a bias neuron in the final layer that is producing the logits. However, when I read the implementation of the two networks, I can see that they include a bias neuron:

https://github.com/Vastlab/vast/blob/d59aed1bd9fd7d4602117513c0da6d332f402369/vast/architectures/architectures.py#L53
https://github.com/Vastlab/vast/blob/d59aed1bd9fd7d4602117513c0da6d332f402369/vast/architectures/architectures.py#L94

Additionally, for visualization in 2D (the flower plot), this bias neuron is very likely to destroy the performance. I have experiences this especially for the Garbage class approach.

I would strongly recommend to remove the bias neuron in the final fully-connected layer.

### PR DESCRIPTION
**Pull request by siebenkopf**
Closes #63

### CODE REVIEW

**Comment by siebenkopf** [comment]
Is anyone willing to review this, or shall I just merge this (it does not change the default behavior).

**Comment by scruz13** [comment]
@siebenkopf while not assigned a reviewer, it looks ok to me :)

**Review by akshay-raj-dhamija — APPROVED** [review]

**Comment by akshay-raj-dhamija** [comment]
Glad to see the repo still being used :)
LGTM! Also removed the reviewer requirement for the repo.
